src/languages/es.ts

```diff
@@ -2310,6 +2310,8 @@ const translations = {
             }),
             settlementFrequency: 'Frecuencia de liquidación',
             deleteConfirmation: '¿Estás seguro de que quieres eliminar este espacio de trabajo?',
+            deleteWithCardsConfirmation:
+                '¿Estás seguro de que quieres eliminar este espacio de trabajo? De este modo, se eliminarán todos los datos de las tarjetas y las tarjetas asignadas.',
```

Comment:
The correct translation is (`de este modo` was redundant):

```suggestion
                '¿Estás seguro de que quieres eliminar este espacio de trabajo? Se eliminarán todos los datos de las tarjetas y las tarjetas asignadas.',
```